packages.json

```diff
@@ -33,6 +33,7 @@
   "https://github.com/abeintopalo/appiconsetgen.git",
```

Comment:
These changes were the output of this [run](https://github.com/SwiftPackageIndex/PackageList/runs/900525444?check_suite_focus=true) and https://github.com/SwiftPackageIndex/PackageList/pull/506